deploy/Dockerfile

```diff
@@ -170,5 +170,13 @@ ENV AGENT_MAX_CONCURRENT_REQUESTS=10
 # This is useful for testing response handling in clusters with duplicated pods.
 ENV AGENT_REQUEST_LIMIT=1000000
 
+# Set to false if we are building the container for a multi-user experience
+# This will disable some agent networks that are not ready/intended for a multi-user
+# experience which would otherwise appear in the manifest.
+# The setting of false here might seem counterintuitive, but what we are really optimizing
+# for is the smoothest possible single-developer experience who will not have this set
+# and whose manifest will have these other networks turned on.
+ENV NOT_FOR_MULTI_USER_EXPERIENCE=false
```

Comment:
New nev var for the Dockerfile build. By default this is set for what we want for a multi-user deployment.
People can still set outside if they really want.

Reply:
Also, regarding the naming: is this really about multi-user experience vs single dev experience? I feel there's some "big picture" name here trying to get out. I know we're "wink wink" targeting MAUI but don't want to have the variable name bound to MAUI. But what is it about these networks that makes them not fit for public/MAUI consumption? Whatever that aspect is, maybe the name should reflect that. Is it something like, "only for local testing by developer who created them"?

I feel like a year from now we'll be scratching our heads about this -- "what does this have to do with multi user?! OHHH it's referring to MAUI!".